src/libstore/globals.hh

```diff
@@ -175,14 +175,21 @@ public:
         getDefaultCores(),
         "cores",
         R"(
-          Sets the value of the `NIX_BUILD_CORES` environment variable in the
-          invocation of builders. Builders can use this variable at their
-          discretion to control the maximum amount of parallelism. For
-          instance, in Nixpkgs, if the derivation attribute
-          `enableParallelBuilding` is set to `true`, the builder passes the
-          `-jN` flag to GNU Make. It can be overridden using the `--cores`
-          command line switch and defaults to `1`. The value `0` means that
-          the builder should use all available CPU cores in the system.
+          Sets the value of the `NIX_BUILD_CORES` environment variable in the [invocation of the `builder` executable](@docroot@/language/derivations.md#builder-execution) of a derivation.
+          The executable can use this variable to control the maximum amount of parallelism.
```

Comment:
What does "the executable" refer to? It's the build process (e.g. some stdenv phases/hooks) that use `NIX_BUILD_CORES`, not a particular executable.

Reply:
It's the `builder` executable from the previous sentence. I didn't want to repeat it in full or say "..., which can use..." since it's a long sentence already.

Note that it says "the executable *can* use this variable". And that is what `stdenv` happens to do, as shown in the example.